linux_os/guide/system/software/integrity/crypto/configure_openssl_crypto_policy/bash/shared.sh

```diff
@@ -2,8 +2,14 @@
 
 OPENSSL_CRYPTO_POLICY_SECTION='[ crypto_policy ]'
 OPENSSL_CRYPTO_POLICY_SECTION_REGEX='\[\s*crypto_policy\s*\]'
+{{% if product in ["fedora", "ol9", "rhel9"] %}}
```

Comment:
For these products, if the `/etc/pki/tls/openssl.cnf` file was previously configured and is technically compliant but using the `=` signal, like this:
`.include = /etc/crypto-policies/back-ends/opensslcnf.config`

The conditional on line 26 will lead to a duplicated configuration, one using `=` and other not using `=`.
Can this cause any trouble or is one line simply ignored? Besides that, would it be better to check both options and only change if none is there or even ensure only the product preference.

Something similar will happen with the Ansible remediation.

Reply:
@marcusburghardt  Thanks for bringing this up.I have modified both Bash and Ansible remediation in a way so that they respect both options (with our without =) in the same way the OVAL check does. Only in case the option is malformed in some other way or missing they remediate. Do you agree with this? Another way would be to modify OVAL instead and use different regex for RHEL 9, OL 9 and Fedora. Both options seem fine to me.

Reply:
The proposed fix is fine. Thanks @vojtapolasek .